frontend/public/components/build.tsx

```diff
@@ -165,6 +167,22 @@ const BuildGraphs = requirePrometheus(({ build }) => {
   );
 });
 
+export const PipelineBuildStrategyAlert: React.FC<BuildsDetailsProps> = ({ obj: build }) => {
+
+  return (
+    <React.Fragment>
```

Comment:
Prefer the shorthand `<>` and `</>`

```suggestion
    <>
```

Reply:
But in the case the fragment isn't needed at all since there's only one element.